tipCalculator.js

```diff
@@ -0,0 +1,38 @@
+$('.plus').click(function(){
+    $('#people').val(Number($('#people').val()) + 1)   ;
+});
+
+$('.minus').click(function(){
+    if($('#people').val() == '1'){
```

Comment:
It is good practice to use the `===` rather than the `==` operator to prevent unwanted type coercion in JS.